internal/libraries/gitutils/gitutils_test.go

```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestResolveTag(t *testing.T) {
+func Test_resolveTag(t *testing.T) {
```

Comment:
I think it's fine leaving it as `TestResolveTag`.

Reply:
OK, I have reverted that change:
https://github.com/arduino/libraries-repository-engine/compare/8d54b1aedf72f227d4a615726a8a772e7bf6d1b6..d29bc2d54fc4edef7b085ac88e1f5c2ee611bb7c
I started using this `Test_` naming scheme for tests of unexported functions because I saw some indications that it was a convention when I researched the subject at that time, but I don't find any official guidance on it other than that `Test` must not be followed by a lowercase character. You're absolutely right to point out that, since the previous test name is perfectly serviceable, it's better to avoid a pointless diff.

I know that it's not generally considered correct to write tests for non-public API, but I still don't fully buy into that rule. More so in a case like this where everything is internal application code for a fairly simple application.